server/lib/src/be/dbvalue.rs

```diff
@@ -651,6 +652,8 @@ pub enum DbValueSetV2 {
     AuditLogString(Vec<(Cid, String)>),
     #[serde(rename = "EK")]
     EcKeyPrivate(Vec<u8>),
+    #[serde(rename = "IM")]
+    Image(Vec<ImageValue>),
```

Comment:
Normally we keep the DB values as seperate enums to proto or anything else, so that if we have to change the proto representation, we keep the DB version stable. This lets us create forward migrations. So I think you need a "DB version" of the ImageValue here.

Reply:
Can do!